api/allUserList/index.ts

```diff
@@ -0,0 +1,32 @@
+import { Request, Response, Router } from 'express'
+import { getConfigValue } from '../configuration'
+import { SERVICES_RD_PROFESSIONAL_API_PATH } from '../configuration/references'
+import * as log4jui from '../lib/log4jui'
+import {exists, valueOrNull} from '../lib/util'
+import { getRefdataAllUserListUrl } from '../refdataAllUserListUrlUtil'
+
+const logger = log4jui.getLogger('user-list')
+
+export async function handleAllUserListRoute(req: Request, res: Response) {
+    try {
+        const rdProfessionalApiPath = getConfigValue(SERVICES_RD_PROFESSIONAL_API_PATH)
+        const response = await req.http.get(getRefdataAllUserListUrl(rdProfessionalApiPath))
+        logger.info('response::', response.data)
+        res.send(response.data)
+    } catch (error) {
```

Comment:
You could have Next and Handle it globally as this will not just fix this endpoint but also all endpoints.

`export async function handleAllUserListRoute(req: Request, res: Response, next: NextFunction) {
...
catch (error) {
next(error)
}
`

Reply:
Please refer how we do it on WebApp here
https://github.com/hmcts/rpx-xui-webapp/tree/master/api/lib/error.handler.ts

Reply:
@Surekha-HMCTS